File: Daily Flash/week 4/day 3/MySolutions/Python/Program4.py

print("\n      ------ Pattern ------ \n")

ch=97
val=2
for i in reversed(range(4)):
	for j in range(0,i+1):
		print("  ",chr(ch),end=" ")
		ch=ch+1
	
	
	ch=ch-val
	val=val-1	
	print("\n")	
	
'''
Output :

(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 3/MySolutions/Python$ python3 Program4.py

      ------ Pattern ------ 

   a    b    c    d 

   c    d    e 

   e    f 

   g 


'''	

File: Daily Flash/week 4/day 3/MySolutions/C/Program4.c
/*
Problem Statement :

Program 4: Write a Program to Print following Pattern.


a  b  c  d
c  d  e 
e  f
g



*/


#include<stdio.h>

void main() {

	printf("\n     --------- Pattern --------- \n\n");
	
	int ch=97;
	int val=2;
	
	for(int i=0;i<4;i++) {
	
		for(int j=4;j>i;j--) {
			
			printf("    %c",ch++);
		}
		
		ch=ch-val;
		val--;	
			
		
		printf("\n\n");
		
	}
	
	printf("\n\n");
}



/*
Output :

(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 3/MySolutions/C$ cc Program4.c 
(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 3/MySolutions/C$ ./a.out

     --------- Pattern --------- 

    a    b    c    d

    c    d    e

    e    f

    g



*/
